jdi-light-mobile-tests/src/main/java/page/objects/jdi/TestSite.java

```diff
@@ -0,0 +1,11 @@
+package page.objects.jdi;
+
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+
+public class TestSite {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestSite' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)